src/app/support-page/support-page.component.pug

```diff
@@ -0,0 +1,38 @@
+.container.main-content
+  h1 Keyboard Shortcuts
+  table
+    thead
+      tr
+        th Action
+        th Keyboard Shortcut
+    tbody
+      tr
+        td Move viewport up
+        td W, Up Arrow
+      tr
+        td Move viewport down
+        td S, Down Arrow
+      tr
+        td Move viewport left
+        td A, Left Arrow
+      tr
+        td Move viewport right
+        td D, Right arrow
+      tr
+        td Zoom / Move viewport home
+        td 0
+      tr
+        td Zoom viewport out
+        td - / _, Shift + S, Shift + Down arrow
+      tr
+        td Zoom viewport in
+        td = / +, Shift + W, Shift + Up arrow
+      tr
+        td Rotate clockwise
+        td R
```

Comment:
We may want to use use a lower case character here and with the other commands where users do not have have to  hit the shift key to perform the action. 

```suggestion
        td r
```

Reply:
Hahaha I use upper case because I see the letters are upper case in the example: https://www.jstor.org/support/text-editor-help/

Reply:
We may want to change the directions to be lowercase there too, but I suggested it here because `w` and `W` perform different actions. This may be a question for Nick though

Reply:
@sandsng can you provide some suggestions?

Reply:
I like the suggestion of switching to lowercase. Makes sense! Thanks!